### ISSUE TITLE
[error] Python server: py4j not on classpath

### ISSUE THREAD

**Issue by LexiconCode** [RaiMan/sikulix4python#4]
Python 32-bit 3.8.7
Java 15 version 15 vm 15.0.1+9 class 59.0 arch 64
`java -jar sikulixapi.jar -p`

`[error] Python server: py4j not on classpath`

Verbose
```
S D:\Backup\Projects\SikuliX\sikulix4python> java -jar sikulixapi.jar -v
[DEBUG]startUp: API: Running: D:\Backup\Projects\voice recognition toolbox\Caster projects\SikuliX\sikulix4python\sikulixapi.jar
[DEBUG]startUp: API: AppData: C:\Users\Main\AppData\Roaming\Sikulix
[DEBUG]startUp: API: starting with classpath: sikulixapi.jar ...
[DEBUG]startUp: API: no extensions.txt nor valid content
[46 debug] RunTime:temp folder ok: C:\Users\Main\AppData\Local\Temp\Sikulix_1318413536
Options: *** options dump
Options: testing =
Options: OptionsFile = C:\Users\Main\AppData\Roaming\Sikulix\SikulixStore\SikulixOptions.txt
Options: *** options dump end
***** show environment for 2.0.4-2020-03-14_08:01 API
user.home: C:\Users\Main
user.dir (work dir): D:\Backup\Projects\SikuliX\sikulix4python
user.name: Main
java.io.tmpdir: C:\Users\Main\AppData\Local\Temp
running 64Bit(amd64) on w (10.0) from a jar
java 15 version 15 vm 15.0.1+9 class 59.0 arch 64
app data folder: C:\Users\Main\AppData\Roaming\Sikulix
executing jar: D:\Backup\Projects\SikuliX\sikulix4python\sikulixapi.jar
*** classpath dump sikulix
  0: sikulixapi.jar
*** classpath dump end
***** show environment end
[DEBUG]startUp: API: *********************** leaving start
[29 debug] Sikulix: starting API
[47 debug] RunTime:temp folder ok: C:\Users\Main\AppData\Local\Temp\Sikulix_273099905
Options: *** options dump
Options: testing =
Options: OptionsFile = C:\Users\Main\AppData\Roaming\Sikulix\SikulixStore\SikulixOptions.txt
Options: *** options dump end
***** show environment for 2.0.4-2020-03-14_08:01 API
user.home: C:\Users\Main
user.dir (work dir): D:\Backup\Projects\SikuliX\sikulix4python
user.name: Main
java.io.tmpdir: C:\Users\Main\AppData\Local\Temp
running 64Bit(amd64) on w (10.0) from a jar
java 15 version 15 vm 15.0.1+9 class 59.0 arch 64
app data folder: C:\Users\Main\AppData\Roaming\Sikulix
executing jar: D:\Backup\Projects\SikuliX\sikulix4python\sikulixapi.jar
*** classpath dump sikulix
  0: D:\Backup\Projects\SikuliX\sikulix4python
  1: sikulixapi.jar
*** classpath dump end
***** show environment end
[287 debug] RunTime:loadlib: trying opencv_java342
[442 debug] RunTime:addToWindowsSystemPath: added to systempath:
C:\Users\Main\AppData\Roaming\Sikulix\SikulixLibs
[444 debug] RunTime:libsExport: folder exists: C:\Users\Main\AppData\Roaming\Sikulix\SikulixLibs (2.0.4 - 202003140801)
[461 debug] RunTime:loadLib: opencv_java342.dll (success)
[461 debug] OCR: start: Tess4J 4.4.1 using Tesseract 4.1.0
SikuliX API: nothing to do
[478 debug] RunTime:***** final cleanup at System.exit() *****
[491 debug] FileManager: deleteFileOrFolder:
C:\Users\Main\AppData\Local\Temp\Sikulix_273099905
[1083 debug] RunTime:***** final cleanup at System.exit() *****
[1106 debug] FileManager: deleteFileOrFolder:
C:\Users\Main\AppData\Local\Temp\Sikulix_1318413536
```

**Comment by RaiMan** [comment]
not available with 2.0.4.

The project is currently paused until later this year.

**Comment by LexiconCode** [comment]
> not available with 2.0.4.
> 
> The project is currently paused until later this year. 

All right! Looking forward to later this year.

**Comment by RaiMan** [comment]
@LexiconCode 
Will have a look, wether I can make it available in 2.0.5 (1st of March) again (but will only be in the experimental state as is in the moment).

... but enough, to make one's own experiences and maybe contribute ideas.

All the best.

**Comment by RaiMan** [comment]
See https://github.com/RaiMan/SikuliX1/issues/419
for further discussion and comments
